
syntax = "proto3";
package myrpc;

message Header{
    bytes server_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
}